conf/nomad/postgres.hcl

```diff
@@ -49,6 +49,13 @@ job "${service_name}" {
       connect {
         sidecar_service {}
       }
+
+      sidecar_task {
+        resources {
+          cpu    = 500
+          memory = 1024
+        }
+      }
```

Comment:
Think the `sidecar_task` need to be inside the ``connect` stanza:
https://github.com/fredrikhgrelland/terraform-nomad-hive/blob/master/conf/nomad/hive.hcl#L40-L61

Reply:
https://github.com/fredrikhgrelland/terraform-nomad-hive/blob/2784d1e222b5beb147a23a4db0248b9c874c25a8/conf/nomad/hive.hcl#L40-L61

Reply:
Yes, you are right. This is changed to:

`      connect {
        sidecar_service {
        }

        sidecar_task {
          resources {
            cpu = "${cpu_proxy}"
            memory = "${memory_proxy}"
          }
        }
      }`